sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -0,0 +1,405 @@
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public EvaluationType(string value) { throw null; }
+        public static Azure.AI.Personalizer.Models.EvaluationType Auto { get { throw null; } }
+        public static Azure.AI.Personalizer.Models.EvaluationType Manual { get { throw null; } }
+        public bool Equals(Azure.AI.Personalizer.Models.EvaluationType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.AI.Personalizer.Models.EvaluationType left, Azure.AI.Personalizer.Models.EvaluationType right) { throw null; }
+        public static implicit operator Azure.AI.Personalizer.Models.EvaluationType (string value) { throw null; }
+        public static bool operator !=(Azure.AI.Personalizer.Models.EvaluationType left, Azure.AI.Personalizer.Models.EvaluationType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct LearningMode : System.IEquatable<Azure.AI.Personalizer.Models.LearningMode>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public LearningMode(string value) { throw null; }
+        public static Azure.AI.Personalizer.Models.LearningMode Apprentice { get { throw null; } }
+        public static Azure.AI.Personalizer.Models.LearningMode LoggingOnly { get { throw null; } }
+        public static Azure.AI.Personalizer.Models.LearningMode Online { get { throw null; } }
+        public bool Equals(Azure.AI.Personalizer.Models.LearningMode other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.AI.Personalizer.Models.LearningMode left, Azure.AI.Personalizer.Models.LearningMode right) { throw null; }
+        public static implicit operator Azure.AI.Personalizer.Models.LearningMode (string value) { throw null; }
+        public static bool operator !=(Azure.AI.Personalizer.Models.LearningMode left, Azure.AI.Personalizer.Models.LearningMode right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class LogsProperties
```

Comment:
Can this be called `LogProperties` instead. That reads better IMO.

Reply:
This is generated from the following endpoint logs/properties. These are the properties of the logs. Autorest autogenerated this class with the name. I do not think we should change the endpoint to log/properties, should I change the name of the autogenerated class ?

Reply:
Yes, changes suggested here are (almost) always for the SDK.
Take a look at [this doc](https://github.com/azure/autorest.csharp#customizing-the-generated-code) to see how to modify the generated code. This case would be [renaming model classes](https://github.com/azure/autorest.csharp#rename-a-model-class).